crates/ruma-client-api/src/discovery/discover_homeserver.rs

```diff
@@ -52,6 +52,11 @@ pub struct Response {
         skip_serializing_if = "Option::is_none"
     )]
     pub authentication: Option<AuthenticationServerInfo>,
+
+    /// Information about the homeserver's trusted proxy to use for sliding sync development.
+    #[cfg(feature = "unstable-msc3575")]
+    #[serde(alias = "org.matrix.msc3575.proxy", skip_serializing_if = "Option::is_none")]
```

Comment:
I haven't added the unstable prefix via a `rename` as this is how it is defined in the MSC as a way to discover the proxy whilst the endpoint is in development: https://github.com/matrix-org/matrix-spec-proposals/blob/b4b4e7ff306920d2c862c6ff4d245110f6fa5bc7/proposals/3575-sync.md#unstable-prefix

Reply:
Not sure if I understood correctly, but I think you want the `rename` while the `alias` one might be debatable.

If you look at the docs for rename[[1]]:

> Serialize and deserialize this field with the given name instead of its Rust name.

While the docs for `alias`[[2]] say:

> Deserialize this field from the given name or from its Rust name.

We want to serialize and deserialize to/from `org.matrix.msc3575.proxy`, which rename achieves.

[1]: https://serde.rs/field-attrs.html#rename
[2]: https://serde.rs/field-attrs.html#alias

Reply:
Whoops, my bad. Even as I was typing this I thought to myself "that seems wrong" but carried on anyway 🤦‍♂️

Updated, thanks.